src/ast/codegen_llvm.cpp

```diff
@@ -2275,10 +2275,32 @@ AllocaInst *CodegenLLVM::getMapKey(Map &map)
       }
       else
```

Comment:
So with this code, you can have arrays as the only key to a map entry. What about support for 2 or more values as a map key?

Reply:
Good point, I'll add support for array used as a part of multi-value key, it should be quite straightforward.